### DISCUSSION TITLE
WORKING SESSION -- 21 DEC @ 8:30am EST / 14:30 CET

### DISCUSSION THREAD

**Discussion by jackiekazil** [mesa/mesa#1918] [General]
Where: [via Matrix](https://matrix.to/#/#project-mesa:matrix.org)

@EwoutH and I are getting together for a working session.  This is open if anyone would like to join, but not required. We will update afterwards and at the next month meeting. This is follow up to the dev session on 16 Dec.

**Main focuses are: MEPs, Experimental Features, and auto deploy are all tied together.** 
We may also discuss: property layer and funding.

**Comment by EwoutH** [comment]
Thanks for setting it up. I added some thoughts on the experimental features in #1909.

**Comment by jackiekazil** [comment]
**Notes**

Repo was created -- https://github.com/projectmesa/meps 
First MEPs Tickets  -- 
[MEP 0 - index](https://github.com/projectmesa/meps/issues/3 ) 
[MEP 1 - What is MEP](https://github.com/projectmesa/meps/issues/2 )
[MEP 2 - Adding Features ](https://github.com/projectmesa/meps/issues/1) -- Related #1909
